index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./css/mainstyles.css">
+    <link rel="stylesheet" href="./css/tabletstyles.css">
+    <link rel="stylesheet" href="./css/mobilecss.css">
+    <title>shelter</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="head">
+            <header>
+                <div class="logo">
+                    <h2 class="title">Cosy House</h2>
+                    <h4 class="subtitle">Shelter for pets in Boston</h4>
+                </div>
+                <ul class="links">
+                    <a class="active" href="">About the shelter</a>
```

Comment:
```suggestion
                    <a class="active" href="#">About the shelter</a>
```

пустые `href` y ссылок невалидны, ставь хотя бы решетку изначально